src/TopPage/DataExtension.php

```diff
@@ -31,7 +31,7 @@ class DataExtension extends BaseDataExtension
      * @var array
      */
     private static $has_one = [
-        'TopPage' => Page::class,
+        'TopPage' => SiteTree::class,
```

Comment:
I'm a bit concerned what will happen when you migrate a page from CMS4 to CMS5 and this relation changes.

I'll create some dummy content block on a CMS4 site and manually hack this relation to reference SiteTree to validate how the Data is migrated.

Reply:
Tested and the value retains its property. Even if it didn't it probably wouldn't be the end of the world since this is meant to act as a sort of cache any way.